### ISSUE TITLE
Error while creating Cognito User Pool Domain

### ISSUE THREAD

**Issue by dristirakeshr** [aws-samples/dcv-web-portal#16]
Error: creating Cognito User Pool Domain (dcv-portal-dev-auth): InvalidParameterException: Domain already associated with another user pool.with module.authentication.aws_cognito_user_pool_domain.main,
│   on modules/authentication/main.tf line 43, in resource "aws_cognito_user_pool_domain" "main":
│   43: resource "aws_cognito_user_pool_domain

However, there is no aws_cognito_user_pool_domain with name dcv-portal-dev-auth

**Comment by signalconductor** [comment]
When I ran into this, it was due to needing to define a unique name for the project. I believe the environment + project name has to be unique across all of AWS so using the defaults fails.

**Comment by roshansys** [comment]
Yes that's fixed the issue. But what happened is after deploy everything it started npm package updates and my instance stop responding. I am using t2.micro for deploying the terraform[deploy.sh] script. can that be an issue?
